i18n/en.json

```diff
@@ -3002,7 +3002,8 @@
   "convert_channel.title": "Convert {display_name} to a Private Channel?",
   "copied.message": "Copied",
   "copy_url_context_menu.getChannelLink": "Copy Link",
-  "copy.block.message": "Copy code block",
+  "copy.code.message": "Copy code",
```

Comment:
This change also changes the message from `Copy code block` to `Copy code` if a piece of code is directly written in channel using ``` (check the screenshot below).

I suppose this change is okay to be consistent across different pages. 

<img width="577" alt="Screen Shot 2022-09-08 at 10 27 42" src="https://user-images.githubusercontent.com/37421564/189074356-93e4caa0-b67d-4f8b-b1b7-94b5fc5dd4c8.png">